.changeset/thirty-mayflies-reply.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
what is this file? 😅

Reply:
its for auto publishing of packages. (will be useful once it is fixed 😂)
all PR now need this file.

Reply:
> * on change it would be nice to see the action in storybook, i dont even see the console log actually

Added action for this 🙌🏻 

> * each name could use like 4px padding left on the span
Fixed 🙌🏻 

> * 2 countries flags fail, prob not your bad
> 
> ![padding](https://user-images.githubusercontent.com/13779759/185091683-c26e587f-2e81-4303-9270-8fdd3dc826b2.png)
> 
> ![broken flag](https://user-images.githubusercontent.com/13779759/185091688-8a2a8cc6-ea31-455e-80ad-9a4117c5ba26.png)
> 
> ![broken flag 2](https://user-images.githubusercontent.com/13779759/185091690-7163fd99-b6e4-4639-8462-09f9b7f69ebb.png)
Yeah no image present for that, can't do anything 😅 

@BillyG83 I guess all the mentioned issues are fixed. 
Also new things added:
1. default index can be set now
2. Custom error and description messages

Validation is pending - probably add it in another PR? this is getting huge would make it hard to review 😅